plugins/microsoft_log_analytics/icon_microsoft_log_analytics/triggers/search/trigger.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        interval = abs(params.get(Input.INTERVAL, 60))
+        interval = abs(params.get(Input.INTERVAL, 900))
```

Comment:
Why this change?

Reply:
According to the plugin.spec.yaml. I didn't notice that I missed that change that in previous PR 😞